packages/nextjs/src/config/wrappers/wrapperUtils.ts

```diff
@@ -12,7 +12,14 @@ declare module 'http' {
   }
 }
 
-function getTransactionFromRequest(req: IncomingMessage): Transaction | undefined {
+/**
+ * Grabs a transaction off a Next.js datafetcher request object, if it was previously put there via
+ * `setTransactionOnRequest`.
+ *
+ * @param req The Next.js datafetcher request object
+ * @returns the Transaction on the request object if there is one, or `undefined` if the request object didn't have one.
+ */
+export function getTransactionFromRequest(req: IncomingMessage): Transaction | undefined {
   return req._sentryTransaction;
 }
```

Comment:
Why did you choose to wrap this line in a function? (Same goes for the setter.)

Reply:
I don't know to be honest. It just made sense to me to put this behind a little bit of abstraction to provide more context why this field exists on the `req` object. Feel free to remove it in the future though! I think the ambient type (`declare module 'http' {` at the top of the file) provides enough context.